### PR TITLE
Fix displaying of block 0 reward and hide previous block <tr>

### PR DESCRIPTION
**Pull request by alexlyp**
This will fix showing value error for block 0.

And we will no longer show previous block -1

### CODE REVIEW

**Comment by cjepson** [comment]
utACK

**Comment by jcvernaleo** [comment]
This doesn't respect units for the block 0 reward.

File: public/views/block.html [hunk 1]
```diff
@@ -59,7 +59,8 @@ <h2 translate>Summary</h2>
           </tr>
           <tr>
             <td><strong translate>Block Reward</strong></td>
-            <td class="text-right text-muted">{{$root.currency.getConvertion(block.reward) || block.reward + ' DCR'}}</td>
+            <td data-ng-show="block.height != 0" class="text-right text-muted">{{block.reward + ' DCR'}}</td>
```
Comment (jcvernaleo) [inline public/views/block.html]: The height check is correct but this probably still needs:
 {{$root.currency.getConvertion(block.reward) || block.reward + ' DCR'}}

File: public/views/block.html [hunk 2]
```diff
@@ -59,7 +59,8 @@ <h2 translate>Summary</h2>
...
+            <td data-ng-show="block.height == 0" class="text-right text-muted">0 DCR</td>
```
Comment (jcvernaleo) [inline public/views/block.html]: And this one maybe:
{{$root.currency.getConvertion(0) || 0 + ' DCR'}}

assuming getConvention doesn't have some weird type requirement.

**Comment by jcvernaleo** [comment]
OK

**Pull request merged**